competencySyncScript

```diff
@@ -45,6 +53,116 @@ function testRequests() {
   }
 }
 
+function getChangedDriveFiles(folders) {
+  var folders = ["testFolder"];
+  var folder = DriveApp.getFoldersByName(folders[0]);
+  
+  while (folder.hasNext()) {
+    var files = folder.next().getFiles();
+    while (files.hasNext()) { //for each file in the folder
+      var file = files.next();
+      var fileData = {
+        "fileName" : file.getName()
+        //"fileDecodedContent" : file.getAs(contentType) //HOW DO I GET AS A STRING???????
```

Comment:
create a function called `getFileAsString()` and put all your logic in there (and possbly a test for it too)

Reply:
Done.